src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -2,60 +2,64 @@
 
 import static java.util.Objects.requireNonNull;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_COUNTRY_CODE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_RATING;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
 
 import seedu.address.logic.CommandHistory;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.person.Person;
+import seedu.address.model.place.Place;
 
 /**
- * Adds a person to the address book.
+ * Adds a place to the address book.
  */
 public class AddCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
-
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
+    public static final String COMMAND_ALIAS = "a";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a place to the address book. "
             + "Parameters: "
             + PREFIX_NAME + "NAME "
-            + PREFIX_PHONE + "PHONE "
-            + PREFIX_EMAIL + "EMAIL "
+            + PREFIX_COUNTRY_CODE + "COUNTRY CODE "
+            + PREFIX_RATING + "RATING "
+            + PREFIX_DESCRIPTION + "DESCRIPTION "
             + PREFIX_ADDRESS + "ADDRESS "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_NAME + "John Doe "
-            + PREFIX_PHONE + "98765432 "
-            + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_NAME + "National University of Singapore "
+            + PREFIX_COUNTRY_CODE + "SGP "
+            + PREFIX_RATING + "4 "
+            + PREFIX_DESCRIPTION
+            + "The National University of Singapore is an autonomous research university in Singapore. "
+            + PREFIX_ADDRESS + "21 Lower Kent Ridge Road, Singapore 119077 "
+            + PREFIX_TAG + "school "
+            + PREFIX_TAG + "studiesHere";
 
-    public static final String MESSAGE_SUCCESS = "New person added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+    public static final String MESSAGE_SUCCESS = "New place added: %1$s";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This place already exists in the address book";
```

Comment:
The constant name should be `MESSAGE_DUPLICATE_PLACE`?

Reply:
Okay